### ISSUE TITLE
[WCAG2.2][Terminal]: Other than dragging, there is no alternate option provided for the users to move the tabs.

### ISSUE THREAD

**Issue by Saiteja341** [microsoft/terminal#16717]
### Windows Terminal version

1.19.10302.0

### Windows build number

26040.1000

### Other Software

Test Environment:
OS: Windows 11 Version 23H2 (OS Build 26040.1000)
App: Windows Terminal 
Screen Reader: Narrator

### Steps to reproduce

1.Open Windows Terminal.
2.Add 4 different tabs.
3.Now try to move the tabs from one position to other using keyboard.
4.Observe the issue.

User Experience:
People with motor impairments who cannot operate content by the only way to actuate a function is by dragging a target element from its initial position to some other position will feel difficulty here, if there is no option provided to user to move the tabs.

Guideline link:

https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements

Attachment:
[Other than dragging,no alternative mechanism is provided to move the tabs..zip](https://github.com/microsoft/terminal/files/14292956/Other.than.dragging.no.alternative.mechanism.is.provided.to.move.the.tabs.zip)

### Expected Behavior

There should be an alternate mechanism to be provided in order to move the tabs from one position to other. 

### Actual Behavior

Other than dragging, no alternative mechanism is provided for the users to move the tabs from one position to other.

**Comment by zadjii-msft** [comment]
@Saiteja341 There is! There's already the `moveTab` actions: [Move tab](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/actions#move-tab)

https://github.com/microsoft/terminal/blob/f30cbef34df5624173a45bdb53667a2cf145f4a9/src/cascadia/TerminalSettingsModel/defaults.json#L387-L388

I'm going to close this one as fixed, since those have been there for a few years now.

**Comment by Saiteja341** [comment]
Closing this, as we can add custom action shortcut for 'Move' from settings.
